cfe_internal/enterprise/federation/federation.cf

```diff
@@ -706,20 +706,28 @@ body contain cfpostgres_user
 
 bundle agent exported_data
 # @brief Run script to dump pg data on feeder hub
+# @note Since version 3.24.0, the inventory view is automatically refreshed by
+# cf-reactor whenever inventory is updated. Hence, you can assume that the
+# inventory is refreshed. However, there is one exception; if the previous
+# refresh happened too recently, the following one will be delayed.
```

Comment:
How would someone know that a refresh was delayed. How long will it be delayed? What's too recently (or where do I go to find that stuff out?